package.json

```diff
@@ -50,8 +50,7 @@
         "uglify-js": "^3.15.2"
     },
     "files": [
-        "build/",
-        "index.d.ts"
```

Comment:
I've removed this file, as it no longer exists since a build step was introduced.